### ISSUE TITLE
Remove 'vfyc' from configuring a solver example 

### ISSUE THREAD

**Issue by akoszegi** [dwavesystems/docs#69]
New users tend to copy paste the configuration in the Getting Started example (https://docs.ocean.dwavesys.com/en/latest/overview/dwavesys.html#configuring-a-d-wave-system-as-a-solver). Since Leap users don't have access to vfyc solvers they get errors when they ping the system. Removing the 'vfyc' option or adding an explanation about the option may reduce the number of users that have issues when they first connect to the system.

**Comment by randomir** [comment]
Actually, @JoelPasvolsky, I would take this and raise it -- let's use defaults for all the fields! Only the token needs to be entered, on average.

This is actually related to [this idea](https://github.com/dwavesystems/dwave-cloud-client/issues/304) of configuration step simplification.

**Comment by JoelPasvolsky** [comment]
For now I'll change to default configs only. @randomir, do you plan to implement such a two-flow config creation? It might be enough to add examples of more advanced config.

**Comment by randomir** [comment]
@JoelPasvolsky, yes I believe we'll have a two-flow configure at some point in future.
